crates/circuit/src/circuit_instruction.rs

```diff
@@ -308,6 +308,13 @@ impl CircuitInstruction {
             .any(|x| matches!(x, Param::ParameterExpression(_)))
     }
 
+    pub fn is_conditioned(&self) -> bool {
```

Comment:
This is largely duplicated with the `condition()` method defined above. You can just do `inst.condition(py).is_some()`. I could see the argument for adding this through, so we don't need the py token or the ref count increment just to check if a condition is present. But I'd argue we should only expose that in rust and not expose it as a pymethod.

Also this is adding a public method to a public class in Python if we were going to keep this it would need a docstring and a release note.

Reply:
Removed in 454208cd9